packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -134,9 +139,34 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 					</div>
 					<div hidden={ ! isOpen }>
 						<div className="plan-item__price-note">
-							{ isFree
-								? __( 'free forever', __i18n_text_domain__ )
-								: __( 'per month, billed yearly', __i18n_text_domain__ ) }
+							{ isFree && __( 'free forever', __i18n_text_domain__ ) }
+							{ ! isFree &&
+								( billingInterval === 'ANNUALLY'
+									? sprintf(
+											__( 'per month, billed as %s annually', __i18n_text_domain__ ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'per month{{br/}}billed yearly' )` *ES Score: 7*